From 9a46afebdc63ba67cd95748f898969d6c622a129 Mon Sep 17 00:00:00 2001
From: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
Date: Wed, 17 Sep 2014 18:08:31 +0100
Subject: [PATCH 1/2] iprutils: Allow CFLAGS to be extended from the environment

We also remove -g, since we don't necessarily want to build with
debugging symbols. Buildroot will add -g if needed.

Based on the former patch by Thomas Petazzoni.

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index c4e2427..66c4942 100644
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ CC = gcc
 
 include version.mk
 
-CFLAGS += -g -Wall $(IPR_DEFINES)
+override CFLAGS += -Wall $(IPR_DEFINES)
 UTILS_VER = $(IPR_MAJOR_RELEASE).$(IPR_MINOR_RELEASE).$(IPR_FIX_LEVEL)
 TAR = cd .. && tar -zcpf iprutils-$(UTILS_VER)-src.tgz --exclude .git --exclude CVS --exclude applied-patches --exclude series --exclude txt --exclude pc --exclude patches --exclude debug --exclude *~* iprutils
 
-- 
1.7.1

